delta-app/src/main/java/io/cdap/delta/app/service/AssessmentHandler.java

```diff
@@ -288,6 +285,88 @@ public void getState(HttpServiceRequest request, HttpServiceResponder responder,
     }));
   }
 
+  /*
+    Methods to manage offset and other states
+   */
+  //Looks up the delta_offset_store table for offset data.
+  @GET
+  @Path("v1/contexts/{namespace}/apps/{app}/generations/{generation}/instanceIds/{instanceid}/offset")
```

Comment:
instanceIds -> instances. Same for other paths

This is because in a REST path, it represents a collection of resources. The resource is an instance and not an id.